source/extensions/filters/http/istio_stats/istio_stats.cc

```diff
@@ -826,38 +827,42 @@ class IstioStatsFilter : public Http::PassThroughFilter,
 
   // AccessLog::Instance
   void log(const Formatter::HttpFormatterContext& log_context,
-           const StreamInfo::StreamInfo& info) override {
+          const StreamInfo::StreamInfo& info) override {
+    // Clear the tags vector to ensure we are not appending to existing tags from a previous log call.
```

Comment:
Are you using the periodic access logger?

Reply:
I am using a stock 1.23 Istio ambient install. I am new to the codebase so I can omit `tags_.clear();` if the logger guarantees a fresh state for each call.

Reply:
log() is only called once normally. However, Envoy added a new periodic reporter that would call log() repeatedly (instead of a custom internal timer). We haven't integrated with that periodic reporting yet.